tools/run_unit_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2021-present Extreme Networks, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Run stratum unit tests inside the stratum dev container.
+# Run from /stratum.
+
```

Comment:
```suggestion
set -e
```
for error propagation?

Reply:
Probably not really needed as there is only one command in this script and it is at the end. But good general practice, yes.